### ISSUE TITLE
Generic.Whitespace.ScopeIndent calculates 0 indent with multi-line comment before scope start

### ISSUE THREAD

**Issue by tscni** [squizlabs/PHP_CodeSniffer#3373]
**Describe the bug**
Generic.Whitespace.ScopeIndent calculates a wrong indentation level (0) if the beginning of the scope starts line is part of a multi-line comment.
The main issue is that a multi-line comment is split into separate tokens per line and the original indentation is not preserved, as the code seems to use the last token before the scope start.

**Code samples**
```php
<?php
if (1) {
    if (2) {
    } /*elseif (3) {
    }*/ elseif (4) {
    } else {
    }
}
```

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs --standard=PSR12 --sniffs=Generic.Whitespace.ScopeIndent test.php`
3. See error message displayed
   ```
   FOUND 2 ERRORS AFFECTING 2 LINES
   -----------------------------------------------------------------------
    6 | ERROR | [x] Line indented incorrectly; expected 0 spaces, found 4
    7 | ERROR | [x] Line indented incorrectly; expected 0 spaces, found 4
   ```
4. Run `phpcbf --standard=PSR12 --sniffs=Generic.Whitespace.ScopeIndent test.php`
5. See formatting result
   ```php
   <?php
   if (1) {
       if (2) {
       } /*elseif (3) {
       }*/ elseif (4) {
   } else {
   }
   }
   ```

**Expected behavior**
No error, no formatting change

**Versions (please complete the following information):**
 - PHP: 7.4
 - PHPCS: 3.6.0
 - Standard: PSR12
